.huskyrc.js

```diff
@@ -1 +0,0 @@
-module.exports = require('kcd-scripts/husky')
```

Comment:
@kentcdodds I removed this to be able to commit. I had an infinite loader that you can see in this little video. (I am using ubuntu 18)
![infiniteLoading](https://user-images.githubusercontent.com/17161484/97812771-03cd9e80-1c84-11eb-9b15-53b6d88cd5b6.gif)
Did you ever heard of any problem of this kind ?

Reply:
`git commit --no-verify` avoids any pre-commit hooks. Any change to the pre-commit hook should be dealt with  in a separate issue/pr.

Reply:
Oh thanks for the command, I totally forgot it exists ^^
I push directly the revert :)
(by the way I didn't expect to merge this change on master, it was only to be able to commit hoping someone will help me like you did ;) )
Otherwise do you have any idea why could the test run infinitely ? (in the hook)

Reply:
No idea. I have husky disabled globally since it doesn't work well with my workflow (commit often, commit failing tests etc).

Reply:
Ahaha no problem :)